corehq/ex-submodules/couchforms/util.py

```diff
@@ -3,11 +3,9 @@
 import hashlib
```

Comment:
i kind of think everything that isn't couch specific should be moved over to `form_processor/utils` and everything that is couch specific should be put in one of the adapters